cmd/lifecycle/analyzer.go

```diff
@@ -45,25 +48,36 @@ type analyzeArgs struct {
 
 func (a *analyzeCmd) DefineFlags() {
 	cmd.FlagAnalyzedPath(&a.analyzedPath)
-	cmd.FlagCacheDir(&a.cacheDir)
 	cmd.FlagCacheImage(&a.cacheImageTag)
-	cmd.FlagGroupPath(&a.groupPath)
+	if a.analyzeLayers() {
+		cmd.FlagCacheDir(&a.cacheDir)
```

Comment:
Should `cmd.FlagCacheImage(&a.cacheImageTag)` go here? The spec PR removes `-cache-image` as a flag.

Reply:
Yes, you are correct. I'll make that change